<launch>
    <group ns="robot_all">
	    <node pkg="reinovo_control" type="blank" name="blank" output="screen" required="true"/>
	</group>
	<!--以上勿动-->

    <!--机器人底盘用-->
        <include file="$(find ar_pose)/launch/ar_base.launch"/>
        <include file="$(find auto_charging)/launch/auto_charging.launch"/>
        <node pkg="oryxbot_interaction" type="oryxbot_navi" name="oryxbot_navi" output="screen"/>

    <!--机器人手臂用-->
        <include file="$(find arm_controller)/launch/pick_ar.launch"/>
        <include file="$(find ar_pose)/launch/ar_pick.launch"/>


</launch>

